src/PublishPort.jsx

```diff
@@ -57,8 +107,14 @@ export const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount })
                             step={1}
                             min={1}
                             max={65535}
+                            validated={validationFailed?.containerPort ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20231009-110933-d1e7ff0e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)